client/extensions/woocommerce/app/settings/email/mailchimp/setup-mailchimp.js

```diff
@@ -0,0 +1,315 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { pick, some, isEmpty } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { localize } from 'i18n-calypso';
+import Dialog from 'components/dialog';
```

Comment:
These imports are not alphabetized - from recent chatter the team likes to have it that was as a standard of sorts.